submissions/evgenii-del/html-css-popup/css/style.css

```
@@ -0,0 +1,296 @@
+body, * {
```

you can remove the selector for the `body` tag
```suggestion
* {
```